charts/pulsar/values.yaml

```diff
@@ -1051,6 +1056,8 @@ broker:
 ##
 functions:
   component: functions-worker
+  useDedicateRunner: true
```

Comment:
should we set to `false` be default?

Reply:
my bad, changed to true for testing, updated

Reply:
done